code/game/gamemodes/game_mode.dm

```diff
@@ -310,10 +310,7 @@
 
 	intercepttext += generate_station_goal_report()
 
-	print_command_report(intercepttext, "Central Command Status Summary", announce=FALSE)
-	priority_announce("A summary has been copied and printed to all communications consoles.\n\n[generate_station_trait_announcement()]", "Enemy communication intercepted. Security level elevated.", ANNOUNCER_INTERCEPT)
-	if(GLOB.security_level < SEC_LEVEL_BLUE)
-		set_security_level(SEC_LEVEL_BLUE)
+	print_command_report(intercepttext, "Central Command Status Summary")
```

Comment:
summary should still have a message telling people about it. possibly just saying "Possible threats located in your region" or something

Reply:
The classified command report already says that. The intercept text contains that information.